cv/CV-guide.md

```diff
@@ -0,0 +1,171 @@
+name: Sampling rates
+definition: "The sampling rates of identified peptides and their respective frequency (accumulated by the sampling rate, i.e. how often peptides were sampled n times and how many peptides were resampled at that rate. Optionally, which peptides were sampled at that rate, space separated)" [PSI:QC]
+comment: The sampling rate may give insights to peptide separation, dynamic exclusion settings and gradient efficiency.
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000009 ! ID based
+is_a: QC:4000006 ! table
+has_column_type: QC:4000089 ! Sample rates
+has_column_type: QC:4000090 ! Counts
+has_optional_column_type: QC:4000091 ! Peptide sequences
+```
+
+Any metric object of the type `Sampling rates` will be expected to contain a column of name `Sample rates`  and `Counts` and may have `Peptide sequences`. These define the respective column values as follows.
+
+```
+[Term]
+id: QC:4000089
+name: Sample rates
+def: "The column contains sample rates as number of occurrences." [PSI:QC]
+is_a: QC:4000082 ! Defined value type
+has_units: PATO:0000161 ! rate
+
+[Term]
+id: QC:4000090
+name: Counts
+def: "The column contains counts of occurrences of a common type." [PSI:QC]
+is_a: QC:4000082 ! Defined value type
+has_units: STATO:0000047 ! count
+
+[Term]
+id: QC:4000091
+name: Peptide sequences
+def: "The column contains peptide sequences." [PSI:QC]
+is_a: QC:4000082 ! Defined value type
+has_units: NCIT:C45253 ! String
+```
+Such, a metric object for two peptides being samples 5 times ("DYHTVLGAR", "AGLVTHYDR") and three peptides sampled 3 times ("IQACSGNDLQQWAR","AWQQLDNGSCAQIR","MATSGMMTELGK") will look like this.
+```
+{
+    "accession": "QC:4000113",
+    "name": "MZ acquisition range",
+    "value": {
+	    "Sample rates": [5,3],
+	    "Counts": [2,3],
+	    "Peptide sequences": ["DYHTVLGAR AGLVTHYDR","IQACSGNDLQQWAR AWQQLDNGSCAQIR MATSGMMTELGK"],
+    },
+}
+```
+
+
+# Define a new metric term for a
+If a desired metric is not yet defined, you will need a cv term definition before you can write a metric object. 
+
+## single value, n-tuple, matrix:
+This is straight forward for these types. 
+1. Make sure, the term is not defined yet in the QC ontology
+2. Define a reasonable descriptive name while keeping concise
+3. Describe the term content, context (relations), and usage 
+4. Define its value type 
+
+Say we want to define a metric for the fastest frequency for MS/MS acquisition. We look up the QC ontology and for the purpose of this document pretend, no such term exists. We coin the name `Maximal MS2 frequency` and define it as `"The fastest frequency for MS/MS collection in any minute over the complete run"`.  This will clearly be a single value type, we do not need any spectrum identifications to assign its value, but it will be for the MS2 tandem spectra of one particular run.  And it will clearly be of type frequency, in particular the sampling frequency of spectra per minute. Hence the might look like this.
```

Comment:
`MS2 tandem spectra ` seems a bit duplicated

`Hence the might look like this.` .. --> `it`?